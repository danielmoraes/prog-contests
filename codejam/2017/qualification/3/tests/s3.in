100
4 2
5 2
6 2
1000 1000
1000 1
2 2
1000 512
1000 469
500 103
500 244
1000 999
3 2
733 708
999 499
470 370
306 238
266 232
349 330
969 811
430 381
722 579
500 500
999 256
999 127
388 366
999 999
500 255
500 249
1000 127
3 1
999 255
4 1
1000 256
600 535
1000 2
999 498
2 1
999 512
294 282
460 440
1000 500
460 412
251 236
657 634
999 511
1 1
1000 499
802 631
408 361
500 1
1000 488
999 998
1000 128
859 812
717 572
500 116
999 497
5 1
289 238
356 307
540 508
500 250
871 798
500 247
999 2
999 505
650 508
546 437
894 835
999 487
504 491
500 245
500 127
500 499
1000 255
961 732
1000 511
1000 502
500 256
410 355
391 350
755 671
973 956
611 581
500 196
903 839
1000 489
500 117
999 488
992 821
999 128
291 246
726 710
1000 498
500 248
500 128
999 1
256 212
500 2
418 393
