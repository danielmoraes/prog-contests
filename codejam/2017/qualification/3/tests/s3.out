Case #1: 1 0
Case #2: 1 0
Case #3: 1 1
Case #4: 0 0
Case #5: 500 499
Case #6: 0 0
Case #7: 0 0
Case #8: 1 1
Case #9: 3 3
Case #10: 1 1
Case #11: 0 0
Case #12: 0 0
Case #13: 0 0
Case #14: 1 0
Case #15: 0 0
Case #16: 0 0
Case #17: 0 0
Case #18: 0 0
Case #19: 0 0
Case #20: 0 0
Case #21: 0 0
Case #22: 0 0
Case #23: 1 1
Case #24: 7 6
Case #25: 0 0
Case #26: 0 0
Case #27: 1 0
Case #28: 1 0
Case #29: 7 6
Case #30: 1 1
Case #31: 3 2
Case #32: 2 1
Case #33: 1 1
Case #34: 0 0
Case #35: 250 249
Case #36: 1 0
Case #37: 1 0
Case #38: 0 0
Case #39: 0 0
Case #40: 0 0
Case #41: 1 0
Case #42: 0 0
Case #43: 0 0
Case #44: 0 0
Case #45: 1 0
Case #46: 0 0
Case #47: 1 0
Case #48: 0 0
Case #49: 0 0
Case #50: 250 249
Case #51: 1 1
Case #52: 0 0
Case #53: 3 3
Case #54: 0 0
Case #55: 0 0
Case #56: 3 3
Case #57: 1 0
Case #58: 2 2
Case #59: 0 0
Case #60: 0 0
Case #61: 0 0
Case #62: 1 0
Case #63: 0 0
Case #64: 1 0
Case #65: 249 249
Case #66: 1 0
Case #67: 0 0
Case #68: 0 0
Case #69: 0 0
Case #70: 1 1
Case #71: 0 0
Case #72: 1 0
Case #73: 3 2
Case #74: 0 0
Case #75: 3 2
Case #76: 0 0
Case #77: 1 0
Case #78: 1 0
Case #79: 0 0
Case #80: 0 0
Case #81: 0 0
Case #82: 0 0
Case #83: 0 0
Case #84: 0 0
Case #85: 1 1
Case #86: 0 0
Case #87: 1 0
Case #88: 3 2
Case #89: 1 0
Case #90: 0 0
Case #91: 3 3
Case #92: 0 0
Case #93: 0 0
Case #94: 1 0
Case #95: 1 0
Case #96: 1 1
Case #97: 499 499
Case #98: 0 0
Case #99: 125 124
Case #100: 0 0
